substrate/client/tests/validator_sets.rs

```diff
@@ -8,11 +8,13 @@ use serai_client::{
     primitives::{Session, ValidatorSet, KeyPair},
     ValidatorSetsEvent,
   },
-  Serai,
+  Amount, Serai,
 };
 
 mod common;
-use common::validator_sets::set_keys;
+use common::validator_sets::{set_keys, allocate_stake, deallocate_stake};
+
+const EPOCH_INTERVAL: u64 = 5;
```

Comment:
We can likely modify our Substrate to take the epoch length as an argument over the CLI. It'd be part of the replace_cmd below (as config is on the node-side, not the runtime-side).